frame/contracts/src/tests.rs

```diff
@@ -4386,8 +4412,8 @@ fn delegate_call_indeterministic_code() {
 
 #[test]
```

Comment:
shouldn't we add a test for the ex. `module imports a non-existent function`, i.e. a case when non-existent host function is being called from the contract (again, quite often case in real world, afaik)

Reply:
There should be something useful in the debug buffer. I will add it to the test so we can see.

Reply:
Ok this doesn't really work as at this point the debug buffer is not available. I improved the docs of the `CodeRejected` error instead.